packages/client/src/generation/TSClient/PrismaClient.ts

```diff
@@ -307,6 +307,22 @@ function runCommandRawDefinition(this: PrismaClientClass) {
   return ts.stringify(method, { indentLevel: 1, newLine: 'leading' })
 }
 
+function applyPendingMigrationsDefinition(this: PrismaClientClass) {
+  const method = ts.method('$applyPendingMigrations').setReturnType(ts.prismaPromise(ts.voidType))
+
+  if (this.runtimeNameTs !== 'rn') {
+    method.setDocComment(ts.docComment`
+  Tries to apply pending migrations to the database.
+  `)
+  } else {
```

Comment:
Rather than telling people not to use and hoping they'll read a doc let's not generate the types for this method at all if it is not react-native build.

Reply:
I tried that, but there was an error and Pierre said you guys needed to merge some changes first. I will give it another go. Is there some sort of target I could access to avoid generating the types?

Reply:
Got it to work I believe, please review. Returning null and later filtering the list of methods to generate